content/en/docs/concepts/overview/components.md

```diff
@@ -56,6 +56,8 @@ Some types of these controllers are:
   * Endpoints controller: Populates the Endpoints object (that is, joins Services & Pods).
   * Service Account & Token controllers: Create default accounts and API access tokens for new namespaces.
 
+*Referred as c-m in fig above.
```

Comment:
I'm not sure this adds clarity or is beneficial in any way. The legend is provided alongside the diagram and people should, ideally, be able to infer from that. I'm in favor of retaining the docs as-is unless somebody else feels that the change is absolutely required.

Reply:
Thank you @divya-mohan0209 for review , but no reference to controller-manager as mentioned in legend in the description
https://kubernetes.io/docs/concepts/overview/components/#kube-controller-manager ( It is always used as kube-controller-manager) so was just trying to link...

Reply:
Thanks , I got you , may be I am nit picking , but I have seen sometimes it causes confusion to newbies if there is any difference in controller manager and kube-controller-manager.

Reply:
No, you're not nitpicking. It's just that I don't specifically find it useful because the cloud controller manager & controller manager are specifically bifurcated. I'll let another reviewer have a look and ascertain if there's any value in adding the line. Thank you for your contribution, nevertheless.

Reply:
I'd be happy to see a **bigger** change to the diagram, eg adding a tooltip for each component, or letting you click on (eg) the k-c-m icon to see details of the component. There _is_ room to improve.

This smaller change feels like it's not enough of an improvement though.

Reply:
very good suggestion , I have no idea on front end as of now .. 

/help 
- If someone can guide or take it up as a new PR , will close this in that case .